cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -981,10 +982,16 @@ protected void heapOptions(List<EnvVar> envVars, double dynamicHeapFraction, lon
             kafkaHeapOpts.append(' ').append("-Xmx").append(xmx);
         } else {
             ResourceRequirements resources = getResources();
-            Map<String, Quantity> cpuMemory = resources == null ? null : resources.getLimits();
+            Map<String, Quantity> cpuMemoryS = resources == null ? null : resources.getRequests();
```

Comment:
While we can discuss what the right setting here is, this is not what the issue suggested ;-)